src/components/model.jsx

```diff
@@ -156,171 +155,208 @@ export default function Model({ app, appLayout }) {
     }
     odd = !odd;
 
-    return (
-      <div key={tableName}>
-        <div
-          className={headerClasses}
-          data-tableheader={tableName}
-          title={getTableTooltip(queryModel.tables[tableName])}
-          style={{
-            margin: 0, zIndex: 3, position: 'sticky', top: 0,
-          }}
-        >
-          <div>{tableName}</div>
-          <div className="nbr-of-rows">{queryModel.tables[tableName].qNoOfRows}</div>
-        </div>
-        <div className={columnClasses} role="tab" tablez={tableName}>
-          <div>
-            {queryModel.resultFieldList.map((fieldName) => {
-              const isFilterboxOpen = openBoxes[fieldName];
-              const cellContainerStyle = {};
-              if (isFilterboxOpen) {
-                cellContainerStyle.height = '24em';
-              }
-
-              const x = queryModel.grid[fieldName][tableName];
-              if (x && !x.isEmpty) {
-                let classes = 'vertcell keycell';
-                if (x.hasAssociationToRight) {
-                  classes += ' hasAssociationToRight';
-                }
-                if (x.hasAssociationToLeft) {
-                  classes += ' hasAssociationToLeft';
-                }
-
-                if (atPlayModel.keysAtPlay[fieldName]) {
-                  classes += ' keyAtPlay';
-                } else if (assocationsHighlighted) {
-                  classes += ' notKeyAtPlay';
-                }
+    const saveTableWidth = (ref, name) => {
+      if (isLocalStorage) {
+        const key = `${app.id}/tables/${name}`;
 
-                const assocStyle = {
-                  backgroundColor: x.backgroundColor,
-                };
+        localStorage.setItem(key, ref.offsetWidth);
+      }
+    };
+
+    const key = `${app.id}/tables/${tableName}`;
+    const minTableWidth = 29 * 8;
+    const savedTableWidth = (isLocalStorage && localStorage.getItem(key));
+    const tableSize = savedTableWidth ? { width: `${savedTableWidth}px` } : 'auto';
+    const handleStyle = {
+      right: {
+        zIndex: 3,
+        width: '3.5em',
```

Comment:
Can this be based off of the `29 * 8` stuff above so it would scale if we tune the numbers?

Reply:
The 3.5 em is connected to the the column padding, and 29 * 8 is sth I totally made up myself 😂 
But I get your message, I'll think of sth!